web/source/kmwosk.js

```diff
@@ -660,19 +700,40 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
         case 'K_SHIFT':
           nextLayer = 'shift'; break;
         case 'K_LCONTROL':
-        case 'K_RCONTROL':
         case 'K_LCTRL':
+          if(keymanweb.isChiral()) {
+            nextLayer = 'leftctrl';
+            break;
+          }
+        case 'K_RCONTROL':
         case 'K_RCTRL':
+          if(keymanweb.isChiral()) {
+            nextLayer = 'rightctrl';
+            break;
+          }
         case 'K_CTRL':
           nextLayer = 'ctrl'; break;
         case 'K_LMENU':
-        case 'K_RMENU':
         case 'K_LALT':
+          if(keymanweb.isChiral()) {
+            nextLayer = 'leftalt';
+            break;
+          }
+        case 'K_RMENU':
         case 'K_RALT':
+          if(keymanweb.isChiral()) {
```

Comment:
Left Alt As per Left Control above (drop through behaviour)

Reply:
Same as mentioned for the Control case.